feature-libs/user/profile/occ/adapters/occ-user-profile.adapter.spec.ts

```diff
@@ -301,28 +300,24 @@ describe('OccUserProfileAdapter', () => {
       const currentPassword = 'Qwe123!';
       const newUserId = 'tester@sap.com';
 
-      let result: Object;
-
       occUserAdapter
         .updateEmail(userId, currentPassword, newUserId)
-        .subscribe((value) => (result = value));
+        .subscribe();
 
       const mockReq = httpMock.expectOne((req) => {
-        return (
-          req.method === 'PUT' &&
-          req.serializeBody() ===
-            `password=${currentPassword}&newLogin=${newUserId}`
-        );
+        return req.method === 'POST' && req.url === '/userUpdateLoginId';
       });
 
       expect(occEndpointsService.buildUrl).toHaveBeenCalledWith(
         'userUpdateLoginId',
         { urlParams: { userId } }
       );
       expect(mockReq.cancelled).toBeFalsy();
-
+      expect(mockReq.request.body).toEqual({
+        newLoginId: newUserId,
+        password: currentPassword,
```

Comment:
```suggestion
        password: currentPassword
```

Reply:
add `,` is demand by prettier fix, otherwise pipeline will have failure